src/actions/playerActions.js

```diff
@@ -44,58 +44,52 @@ export const getDevices = () => dispatch => {
     .catch(e => console.log(e));
 };
 
-export const getCurrentPlaying = () => dispatch => {
-  playerService
-    .getCurrentlyPlaying()
-    .then(track => {
-      dispatch({ type: GET_CURRENT_PLAYING, payload: track });
-    })
-    .catch(e => console.log(e));
+export const getCurrentlyPlaying = () => async dispatch => {
+  const track = await playerService.getCurrentlyPlaying();
+  dispatch({ type: GET_CURRENT_PLAYING, payload: track });
+  // .then(track => {
```

Comment:
Remove this kind of comments if you don't need the code anymore

Reply:
ok!